lib/wallaby/feature_case.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Wallaby.FeatureCase do
+  use ExUnit.CaseTemplate
+
+  @otp_app Application.get_env(:wallaby, :otp_app)
+
+  using do
+    quote do
+      ExUnit.Case.register_attribute(__MODULE__, :sessions)
+
+      use Wallaby.DSL
+      import Wallaby.FeatureCase
+    end
+  end
+
+  setup context do
+    sessions_count = get_in(context, [:registered, :sessions]) || 1
+    capabilities = Application.get_env(:wallaby, :capabilities)
+
+    metadata =
+      @otp_app
+      |> ecto_repos()
+      |> Enum.map(&checkout_ecto_repos(&1, context[:async]))
+      |> metadata_for_ecto_repos()
+
+    sessions =
+      1..sessions_count
+      |> Enum.map(fn _ ->
+        context = String.replace(to_string(context.test), " ", "_")
```

Comment:
It took me a while to see that `session.context` is just modified test message and not the whole context. Is there a different name we could use?

Also if this will be used in screenshot file name, I think there may be some illegal characters in test message, we could try to handle that

Reply:
I think renaming is an okay option. I just followed the naming theme from #156 without thinking about it much.

I will look into the illegal characters.